src/foam/dao/TTLCachingDAO.js

```diff
@@ -23,7 +23,11 @@ foam.CLASS({
     {
       /** The cache to read items quickly. */
       name: 'cache'
-    }
+    },
+    {
+      class: 'Long',
+      name: 'purgeTime'
+    },
```

Comment:
```suggestion
      value: 15000
    },
```

Reply:
Should default value for purgeTime be removed from EasyDAO then?